### ISSUE TITLE
Feature Request: Message Read indicator

### ISSUE THREAD

**Issue by kyle0440** [osTicket/osTicket-1.8#2231]
I think it would be helpful if staff could see if their messages had been viewed by the end user and, if so, when. 

(I realize this wouldn't work for responses sent and read by the end user via email, but the emails that get sent could always be modified to not include the actual message and just a link instead, at the admin's discretion.)

**Comment by greezybacon** [comment]
Fair enough. Most systems use a remote image in the email that calls the help desk back and is used to confirm the read. I think we'll get there eventually
